programming/sed-article.md

```diff
@@ -246,17 +234,17 @@ followed by characters that are not the ending delimiter,
 followed by the ending delimiter.
 For example, to match zero or more characters surrounded by parentheses,
 use `\([^)]*\)`.
-Note that parentheses outside the character class
-must escaped with a backslash,
-but parentheses inside a character class are not.
+Note that the parentheses outside the character class
+are escaped with a backslash while the paren inside the character class is not escaped.
 
 The `?`, `*`, and `+` repetition characters are greedy.
 This means the entire regular expression will
 match as many characters as possible.
+"Greediness" is important for grouping which will be described later.
 
 Suppose we want to match an underscore, followed by
 any other characters, and another underscore.
-An example is "_apple_banana_cherry_".
+An example is "\_apple\_banana\_cherry\_".
```

Comment:
Are the added backslashes only for the benefit of markdown?

Reply:
Yes.  It italicizes banana otherwise.

Reply:
MarkdownLint says I only need to escape the first and last underscores. When I do that, nothing inside is italicized.